### ISSUE TITLE
req.swagger.params not defined

### ISSUE THREAD

**Issue by mhamill** [apigee-127/swagger-express#15]
The latest version of swagger-express-mw no longer defines the req.swagger.params object. This was not an issue with version 0.1.0, which I was on before.

I also noticed that running swagger project install still installs version 0.1.0 of this software.

**Comment by mhamill** [comment]
It appears that this issue is related to a breaking change documented at https://github.com/theganyo/swagger-node-runner/releases/tag/v0.6.0.

If my understanding is correct, to get req.swagger.params working again, I need to add the swagger_params_parser line into default.yaml. Can anyone confirm please?

**Comment by theganyo** [comment]
Yes. See also this: https://github.com/apigee-127/swagger-express/issues/4
